src/engagement_db_to_analysis/code_imputation_functions.py

```diff
@@ -74,12 +81,25 @@ def _impute_age_category(user, messages_traced_data, analysis_dataset_configs):
                 Metadata(user, Metadata.get_call_location(), TimeUtils.utc_now_as_iso_string())
             )
 
-            messages_traced_data[message_index] = message
+            age_messages +=1
+            imputed_labels +=1
```

Comment:
I think this is confusing because if we're tracking two variables then surely they should be tracking different things/be able to reach different numbers?

I think the numbers we care about are: (i) how many messages did you look at, and (ii) how many got imputed

Reply:
This was more of a validation log that we have imputed the same number of labels for the available age messages. Have moved age_messages += 1 to increment before imputing
        `if message["dataset"] in age_engagement_db_datasets:
            age_messages += 1`
but happy to log the number of messages traced data if you think that's more useful.